GETTING-STARTED.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started
+
+## Setup and Installation
+
+### Prerequisites
+
+In order to use **Django Deploy**, you must first install the following dependencies:
+- [Python](https://www.python.org/downloads/) 3.5 or higher
+- [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+- [Docker](https://docs.docker.com/install/overview/) (any edition)
+- [Google Cloud SDK](https://cloud.google.com/sdk/docs/quickstarts)
+- [kubectl](https://cloud.google.com/kubernetes-engine/docs/quickstart) (can be
+  installed with `gcloud components install kubectl`)
+- [Cloud SQL Proxy](https://cloud.google.com/sql/docs/mysql/connect-admin-proxy#install)
```

Comment:
Our tool can automatically install Cloud SQL Proxy, should we still mention it in the prerequisites?